### ISSUE TITLE
Which time value is reported?

### ISSUE THREAD

**Issue by sradc** [KellerJordan/cifar10-airbench#5]
Hey, I'm a fan of your work, here and modded-NanoGPT. Quick question:

The readme reports the mean accuracy, which I understand is the mean of the accs from the 200 random weight initialisations.

```python
accs = torch.tensor([main(run, model) for run in range(200)])
print("Mean: %.4f    Std: %.4f" % (accs.mean(), accs.std()))
```

What is reported for the time? Is it the mean / min / median? (I couldn't see this from the [paper](https://arxiv.org/abs/2404.00498) either.)

Is it the case that you see so little variance in time that it's easy to report a typical run from the printouts?

**Comment by KellerJordan** [comment]
it's the min/best time
